packages/neuron-ui/src/components/LockInfoDialog/index.tsx

```diff
@@ -14,7 +15,38 @@ interface LockInfoDialogProps {
   onDismiss: () => void
 }
 
-const ShortAddr = ({ lockScript, isMainnet }: { lockScript: CKBComponents.Script | null; isMainnet: boolean }) => {
+const useCopy = () => {
+  const timer = useRef<ReturnType<typeof setTimeout>>()
+  const [copied, setCopied] = useState(false)
+  const [refreshKey, setRefreshKey] = useState(1)
```

Comment:
What does `refresh key` do, only for `key` of duplicated components?

Reply:
Add `key` to force to re-render the `Alert` component. Because the `Alert` has animation. Otherwise, if there is no change in the component when the user clicks copy more times in an animation execute time, the animation will not normally. I can also use `debounce` to show the last animation.